### PR TITLE
Feature/DTGB-214: Some button fixes and doc changes.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Comment by meirege** [comment]
@delrueba Can you take a final look at the buttons from an accessibility point of view?

File: styleguide/components/00-settings/_vars.scss [hunk 1]
```diff
@@ -182,7 +182,7 @@ $default-font-size: 20px !default; //px
 /// @access public
 /// @author Gert-Jan Meire
 ///
-$default-line-height: 1.4em !default; //em
+$default-line-height: 1.75 !default;
```
Comment (delrueba) [inline styleguide/components/00-settings/_vars.scss:185]: :+1:

File: styleguide/components/21-atoms/buttons/button-alert/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Alert button variant
+
+## Web accessibility
+
+* Button elements trigger an action and always look like buttons.
+* Link elements trigger navigation and look like links.
+* If links receive a button style, the distinction between button elements
+  is conveyed visually.
```
Comment (delrueba) [inline styleguide/components/21-atoms/buttons/button-alert/README.md:8]: Is there a ticket for this? 
I haven't seen it in the designs yet.

**Review by delrueba — COMMENTED** [review]
OK, if we don't forget the ticket for link-buttons

**Review by delrueba — APPROVED** [review]

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**